File: platforms/web/examples/scene/material/occlusion_texture.js
import { Material, Renderer } from "fragmentcolor";

const renderer = new Renderer();
const ao_bytes = [ 220, 0, 0, 255, 180,   0, 0, 255, 200,   0, 0, 255, 160,   0, 0, 255, ];
const ao = await renderer.createTexture(ao_bytes, [2, 2]);
const mat = Material.pbr().occlusionTexture(ao);

File: platforms/python/examples/scene/material/occlusion_texture.py
from fragmentcolor import Material, Renderer

renderer = Renderer()
ao_bytes = [
    220, 0, 0, 255,
    180,   0, 0, 255,
    200,   0, 0, 255,
    160,   0, 0, 255,
]
ao = renderer.create_texture(ao_bytes, size=[2, 2])
mat = Material.pbr().occlusion_texture(ao)